src/main/java/com/github/seanroy/plugins/DeployLambdaMojo.java

```diff
@@ -119,8 +119,8 @@ private boolean shouldUpdate(LambdaFunction lambdaFunction, GetFunctionResult ge
                 .withDescription(lambdaFunction.getDescription())
                 .withHandler(lambdaFunction.getHandler())
                 .withRole(lambdaRoleArn)
-                .withTimeout(timeout)
-                .withMemorySize(memorySize)
+                .withTimeout(ofNullable(lambdaFunction.getTimeout()).orElse(timeout))
```

Comment:
Nice catch! .withTimeout(lambdaFunction.getTimeout()) is enough as it is defaulted already in AbstractLambdaMojo:221